src/functional/ffront/foast_to_itir.py

```diff
@@ -467,19 +467,18 @@ def _visit_type_constr(self, node: foast.Call, **kwargs) -> itir.Literal:
         raise FieldOperatorLoweringError(f"Encountered a type cast, which is not supported: {node}")
 
     def _make_literal(self, val: Any, type_: ct.ScalarType) -> itir.Literal:
```

Comment:
same here, type_ should also include ct.TupleType as an option

Reply:
Good catch.